cmd/pop/cli/start.go

```diff
@@ -269,6 +269,13 @@ func setupRepo() (string, bool, error) {
 				Default: "5",
```

Comment:
Can you update that one too: "Max price per byte in attoFIL". Let's try to stay consistent across all prompts